src/components/benchmarks/Benchmarks.tsx

```diff
@@ -26,6 +26,11 @@ export default class Benchmarks extends React.Component<
   public render() {
     return (
       <>
+        <div className="btn-layout">
+          {/* operations */}
+          <button className="button-operations btn btn-success">Start</button>
```

Comment:
Are these buttons for force-start and stop modules? @Harkishen-Singh

Reply:
they are there just for the name. We were taking screenshots so kept it there for the view. For now, let it be and someone else can link it with the message to trigger the `start` or `stop`

Reply:
Yeah, I was just asking for purpose. Sure!